Mathlib/MeasureTheory/Constructions/EventuallyMeasurable.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2024 Felix Weilacher. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Felix Weilacher
+-/
+
+import Mathlib.MeasureTheory.MeasurableSpace.Defs
+import Mathlib.Order.Filter.CountableInter
+
+/-!
+# Measurability modulo a filter
+
+In this file we consider the general notion of measurability modulo a σ-filter.
+Two important instances of this construction are null-measurability with respect to a measure,
+where the filter is the collection of co-null sets, and
+Baire-measurability with respect to a topology,
```

Comment:
I don't think you are currently defining this in terms of `EventuallyMeasurableSpace`, right? Perhaps you might want to consider adding it in this PR, or mention that this will be added in the future?

Reply:
What do you mean? This PR does redefine null measurability. Baire measurability is not in mathlib at all yet, but the PR description states this as the next goal.

Reply:
I'm sorry, I only meant to comment on line 16. I meant that the PR description won't be visible in Mathlib, so it might perhaps be good to incorporate this in the docstring for now, to indicate that Baire measurability will be added in the near future. I was trying to avoid the situation where people might think we have Baire measurability based on this comment.

Reply:
I see, thanks